.github/workflows/buildpack-integration-test.yml

```diff
@@ -11,11 +11,13 @@ permissions: read-all
 
 jobs:
   ruby30-buildpack-test:
-    uses: GoogleCloudPlatform/functions-framework-conformance/.github/workflows/buildpack-integration-test.yml@v1.8.0
+    uses: GoogleCloudPlatform/functions-framework-conformance/.github/workflows/buildpack-integration-test.yml@garethgeorge/robust-logs
```

Comment:
Should we release a new patch instead of referencing a branch?

Reply:
Sorry kenneth -- thanks for taking a look at this one. This PR should have been marked as a draft, the buildpack integration tests are actually still broken. The branch pinning is because I've updated the conformance tests as well to add visibility into the failure as I debug the problem.